vm-setup/roles/libvirt/templates/ironic_nodes.json.j2

```diff
@@ -20,16 +20,15 @@
     {% set vm_driver_tmp = vm_driver -%}
     {# If vm_driver == mixed we alternate between ipmi and refish to test both #}
     {% if vm_driver == 'mixed' -%}
-      {# For live-iso setting one node to always use redfish-virtualmedia #}
-      {% if 'redfishvmnode' in node.name -%}
-        {% set vm_driver_tmp = 'redfish-virtualmedia' -%}
       {# For extraworker always use redfish #}
-      {% elif 'extraworker' in node.name -%}
+      {% if 'extraworker' in node.name -%}
         {% set vm_driver_tmp = 'redfish' -%}
-      {% elif loop.index is divisibleby 2 -%}
+      {% elif loop.index % 3 == 0 -%}
```

Comment:
I checked the bmh manifests, and it is always ipmi/redfish-virtualmedia combination when NUM_NODES=2, leaving out redfish. That is because jinja template when `loop.index` is used starts the index with 1 and not 0. If you want it to start the index with 0, then you have to change it to `loop.index0`, xref: https://stackoverflow.com/a/21459009

Reply:
fixed